news/4732.feature

```diff
@@ -0,0 +1,2 @@
+Add --hashes option to pip freeze, to allow pip-formatted output with hashes included.
```

Comment:
We use double back-ticks to surround added command line flags in news. e.g. "\`\`hashes\`\`" (also used in https://github.com/pypa/pip/blob/master/news/8408.removal for e.g.)

Reply:
done